src/System.Net.Http/src/System/Net/Http/Headers/HttpHeaders.cs

```diff
@@ -206,8 +206,11 @@ public bool TryGetValues(string name, out IEnumerable<string> values)
             return TryGetValues(descriptor, out values);
         }
 
+        protected virtual void EnsureHeaders() { }
```

Comment:
See previous comment/question... can we just make this internal or private protected for now?